django_date_extensions/fields.py

```diff
@@ -172,20 +221,27 @@ def clean(self, value):
         if isinstance(value, ApproximateDate):
             return value
         value = re.sub('(?<=\d)(st|nd|rd|th)', '', value.strip())
-        for format in settings.DATE_INPUT_FORMATS:
+        for date_format in settings.DATE_INPUT_FORMATS:
```

Comment:
[format](https://docs.python.org/2/library/functions.html#format) is a built-in function hence renaming it.